lib/active_merchant/billing/gateways/merchant_e_solutions.rb

```diff
@@ -94,6 +97,14 @@ def void(transaction_id, options = {})
         commit('V', nil, options.merge(post))
       end
 
+      def verify(credit_card, options = {})
+        post = {}
+        money = options[:verify_amount].to_i || 0
```

Comment:
[Gateway documentation ](https://anypoint.mulesoft.com/exchange/portals/merchant-e-solutions-3/062a2db4-e8c3-4086-929c-e8e59f8a6ff7/mes-gateway-sapi_v1/minor/0.0/console/documentation/%23145/)says the `transaction_amount` value on verification should always be 0, so I don't think we want to give the option of passing a `verify_amount` here.

Reply:
yes that makes more sense. removing the `verify_amount` from options.